frontend/src/components/ProfileSidebar.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Grid, Avatar, Typography } from '@material-ui/core';
+import PersonIcon from '@material-ui/icons/Person';
+import NotificationsIcon from '@material-ui/icons/Notifications';
+import SettingsIcon from '@material-ui/icons/Settings';
+import MailIcon from '@material-ui/icons/Mail';
+import LockIcon from '@material-ui/icons/Lock';
+
+// ProfileSidebar props.
+type Props = {
+    name: string;
+    profileImg: string;
+}
```

Comment:
I think this should have a more descriptive name than `Props`. Why not `ProfileSidebarProps`?